pkg/rotator/rotator_suite_test.go

```diff
@@ -32,13 +33,13 @@ func TestMain(m *testing.M) {
 }
 
 // StartTestManager adds recFn
-func StartTestManager(mgr manager.Manager, g *gomega.GomegaWithT) (chan struct{}, *sync.WaitGroup) {
-	stop := make(chan struct{})
+func StartTestManager(mgr manager.Manager, g *gomega.GomegaWithT) (context.Context, context.CancelFunc, *sync.WaitGroup) {
```

Comment:
Would it be cleaner to pass a context to this function instead of a creating it here?

Reply:
Looking in some more detail there is already a context created in the test func that is already used. So I might just pass that to this func. 
WDYT?

Reply:
My guess is that Oren would be happy with that :)

Reply:
Super happy :)